stubs/Symfony/Contracts/Cache/CacheInterface.stub

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Symfony\Contracts\Cache;
+
+interface CacheInterface
+{
+    /**
+     * @template T
+     *
+     * @param \Symfony\Contracts\Cache\CallbackInterface<T>|callable(\Psr\Cache\CacheItemInterface): T $callback
```

Comment:
`\Psr\Cache\CacheItemInterface=`, notice the `=` at the end. See https://phpstan.org/writing-php-code/phpdoc-types#callables.

For the `Parameter $callback of method Symfony\Contracts\Cache\CacheInterface::get() has invalid type Psr\Cache\CacheItemInterface.` error, we need to actually add the `Psr\Cache\CacheItemInterface` stub as well. It should be sufficient for it to be an empty class. We only need to know the class, we don't care about methods. See https://phpstan.org/user-guide/stub-files, `Why is PHPStan complaining about “Class not found” in a stub file?`

Reply:
Thanks for the feedback, I was not familiar with the `= optional` notation, actually thought it was a typo.